compiler/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
 
     // Gives us access to annotations.
     implementation libs.kotlin.inject.runtime
+    implementation libs.anvil.ksp.annotations
```

Comment:
I'd like to make this integration totally optional, including the dependencies by using a `compileOnly` dependency here.